src/styles.css

```diff
@@ -145,8 +145,11 @@ button {
 #prompt1,
 #prompt2 {
   opacity: 0;
-  color: red;
+  color: white;
+  background-color: red;
```

Comment:
- [optional] Great job following the wireframe and adding your design. Please make the text bolder and buttons a little bigger, so it can be easily accessed. This will make it more user-friendly.